build_msvc/bitcoin-config.h

```diff
@@ -0,0 +1,428 @@
+/* src/config/bitcoin-config.h.  Generated from bitcoin-config.h.in by configure.  */
+/* src/config/bitcoin-config.h.in.  Generated from configure.ac by autoheader.  */
+
+#ifndef BITCOIN_CONFIG_H
+
+#define BITCOIN_CONFIG_H
```

Comment:
You might have to update the linter to avoid spurious warnings:

```diff
diff --git a/contrib/devtools/lint-include-guards.sh b/contrib/devtools/lint-include-guards.sh
index 6a0dd556bb..d9cf8effee 100755
--- a/contrib/devtools/lint-include-guards.sh
+++ b/contrib/devtools/lint-include-guards.sh
@@ -9,7 +9,7 @@
 HEADER_ID_PREFIX="BITCOIN_"
 HEADER_ID_SUFFIX="_H"
 
-REGEXP_EXCLUDE_FILES_WITH_PREFIX="src/(crypto/ctaes/|leveldb/|secp256k1/|tinyformat.h|univalue/)"
+REGEXP_EXCLUDE_FILES_WITH_PREFIX="(build_msvc/|src/(crypto/ctaes/|leveldb/|secp256k1/|tinyformat.h|univalue/))"
 
 EXIT_CODE=0
 for HEADER_FILE in $(git ls-files -- "*.h" | grep -vE "^${REGEXP_EXCLUDE_FILES_WITH_PREFIX}")
```

Warning was:

```
build_msvc/bitcoin-config.h seems to be missing the expected include guard:
  #ifndef BITCOIN_BITCOIN-CONFIG_H
  #define BITCOIN_BITCOIN-CONFIG_H
  ...
  #endif // BITCOIN_BITCOIN-CONFIG_H
^---- failure generated from contrib/devtools/lint-include-guards.sh